scratch/test_make_growth_rate_plots.R

```diff
@@ -0,0 +1,149 @@
+# Load in DE's 2023-010-16 input data
+library(ggplot2)
+library(zoo)
+input_hosp_data <- readr::read_tsv("output/input_hosp_data.tsv")
```

Comment:
Yeah I don't think there's even really a "user" here other than me tbh. I can just delete this file! I find it easier to do some EDA stuff with an example in a script and then convert to a function, so that's all I was doing here

Reply:
That makes sense, but I was a little sad I couldn't demo by just running!

I think add to gitignore so you can hang onto your test script but I think its overall better not to have code that doesn't run.

Another possibility is to add a function to get some example data; then keep the scripts as examples (maybe add to docs). That would be a new issue for me.